frontend/src/setupProxy.js

```diff
@@ -3,14 +3,14 @@
 const proxy = require('http-proxy-middleware');
 
 let server_url = process.env.REACT_APP_SERVER_URL || 'localhost';
-let server_port = process.env.REACT_APP_FRONT_END_PORT || '5000';
+let server_port = process.env.REACT_APP_FRONT_END_PORT || '5050';
```

Comment:
This change was supposed to be only locally right?

Reply:
yes holy smokes